internal/api/api_client.go

```diff
@@ -21,18 +22,22 @@ func (e ClientError) Error() string {
 	return string(e)
 }
 
-func NewApiClient(httpClient HTTPClient) *ApiClient {
+func NewApiClient(httpClient HTTPClient, siriusUrl string, backendUrl string) (*ApiClient, error) {
 	return &ApiClient{
-		http: httpClient,
-	}
+		http:       httpClient,
+		siriusUrl:  siriusUrl,
+		backendUrl: backendUrl,
+	}, nil
 }
 
 type HTTPClient interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
 type ApiClient struct {
```

Comment:
this is `api.ApiClient`, it would read better named `api.Client` (and then `api.NewClient` above)

Reply:
done